docs/blog/moving-portland-prague-2020-online.rst

```diff
@@ -19,9 +19,9 @@ Your Portland conference ticket
 
 If you currently have a ticket to Write the Docs Portland, your options are:
 
-- Pre-order a ticket to the online event for $100 and get the rest of your original ticket amount refunded.
-- Pre-order a ticket to the online event for $100 and contribute the rest of your original ticket amount to the Write the Docs organization. We are a self-sustaining community and our events are our main source of income, so your contributions help ensure that we can continue to organize events and maintain the global community in years to come.
-- Get a full refund of your original ticket amount on **May 10th, 2020**. This option will happen automatically if you don't select either of the previous options.
+- Pre-order a ticket to the online event for 25% and get the rest of your original ticket amount refunded.
```

Comment:
```suggestion
- Pre-order a ticket to the online event for 25% of the original value of your ticket and get the rest refunded.
```
I think this framing is clearer. Can we be clear about which side of the fees this falls (I assume we take those)

Reply:
And smiliar for all tickets

Reply:
I thought the price was going to be $100. 

25% of $500 = $125. Is this intentional, or did you mean 20%?

Reply:
Yes, a % allows us to offer scaled pricing based on the value of the ticket, like normal.